pkg/operator/ceph/cluster/osd/envs.go

```diff
@@ -213,6 +213,19 @@ func osdActivateEnvVar() []v1.EnvVar {
 	return append(cephVolumeEnvVar(), monEnvVars...)
 }
 
+func getEnvFromSources() []v1.EnvFromSource {
+	optionalConfigMapRef := true
+
+	return []v1.EnvFromSource{
+		{
+			ConfigMapRef: &v1.ConfigMapEnvSource{
+				LocalObjectReference: v1.LocalObjectReference{Name: "rook-ceph-daemon-env-override"},
```

Comment:
Let's use a constant for `rook-ceph-osd-env-override`.

Reply:
Reworked.